### PR TITLE
Add WebStorage example

### PR DESCRIPTION
**Pull request by illia-svystunov**
Example for feature TeamDev-IP/JxBrowser#2351

Add example that demonstrates how to use `WebStorage` API.

### CODE REVIEW

**Comment by illia-svystunov** [comment]
@vladimir-ikryanov PTAL

File: webstorage/src/main/java/WebStorageApi.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import com.teamdev.jxbrowser.engine.EngineOptions;
+import com.teamdev.jxbrowser.frame.WebStorage;
+import com.teamdev.jxbrowser.navigation.event.FrameLoadFinished;
+
+import static com.teamdev.jxbrowser.engine.RenderingMode.OFF_SCREEN;
+
+/**
+ * This example demonstrates how to access the local/session storage and perform operations with it.
+ */
+public class WebStorageApi {
```
Comment (vladimir-ikryanov) [inline webstorage/src/main/java/WebStorageApi.java]: Make it `final`.

File: webstorage/src/main/java/WebStorageApi.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    public static void main(String[] args) {
+        Engine engine = Engine.newInstance(
+                EngineOptions.newBuilder(OFF_SCREEN).build());
+        Browser browser = engine.newBrowser();
+
+        browser.navigation().on(FrameLoadFinished.class, event -> {
+            WebStorage webStorage = event.frame().localStorage(); // localStorage access
+//            WebStorage webStorage = event.frame().sessionStorage(); // sessionStorage access
```
Comment (vladimir-ikryanov) [inline webstorage/src/main/java/WebStorageApi.java]: Do not commit "dead code".

File: webstorage/src/main/java/WebStorageApi.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            String key = "your-custom-key";
+            webStorage.putItem(key, "Hello!");
+            System.out.println(webStorage.item(key).orElseThrow(IllegalStateException::new));
```
Comment (vladimir-ikryanov) [inline webstorage/src/main/java/WebStorageApi.java]: I suggest that you show how JavaScript can read the item you put through JxBrowser API.

File: webstorage/src/main/java/WebStorageApi.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+    public static void main(String[] args) {
+        Engine engine = Engine.newInstance(
+                EngineOptions.newBuilder(OFF_SCREEN).build());
+        Browser browser = engine.newBrowser();
+
+        browser.navigation().on(FrameLoadFinished.class, event -> {
+            WebStorage webStorage = event.frame().localStorage(); // localStorage access
+//            WebStorage webStorage = event.frame().sessionStorage(); // sessionStorage access
+            String key = "your-custom-key";
```
Comment (vladimir-ikryanov) [inline webstorage/src/main/java/WebStorageApi.java]: Use concrete key name and value.

File: webstorage/src/main/java/WebStorageApi.java [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+            webStorage.putItem(key, "Hello!");
+            System.out.println(webStorage.item(key).orElseThrow(IllegalStateException::new));
+        });
+
+        browser.navigation().loadUrl("https://www.teamdev.com");
```
Comment (vladimir-ikryanov) [inline webstorage/src/main/java/WebStorageApi.java]: I think you do not need to load a web page to show how to work with web storage.

**Comment by illia-svystunov** [comment]
@vladimir-ikryanov PTAL

**Pull request merged**